test/prism/ripper_test.rb

```diff
@@ -4,6 +4,42 @@
 
 module Prism
   class RipperTest < TestCase
+    # Ripper produces certain ambiguous structures. For instance, it often
+    # adds an :args_add_block with "false" as the block meaning there is
+    # no block call. It can be hard to tell which of multiple equivalent
+    # structures it will produce. This method attempts to return a normalized
+    # comparable structure.
+    def normalized_sexp(parsed)
```

Comment:
I'm fine having this for now, but we'll definitely want to get rid of this before we close this ticket out.

Reply:
I'll definitely try. I'm having trouble figuring out when Ripper does and doesn't emit this, even looking through parse.y.